tests/js/tooltipTests.js

```diff
@@ -49,6 +24,10 @@ https://github.com/gpii/universal/LICENSE.txt
         tooltipContentMap: {
             "button1": "button1Label",
             "button2": "button2Label"
+        },
+        distributeOptions: {
```

Comment:
Isn't `distributeOptions` already included in the grade `gpii.firstDiscovery.msgLookup`?